lib/oc-tests/test_shareDir.py

```diff
@@ -95,6 +95,13 @@ def setup(step):
     check_users(config.oc_number_test_users)
 
     reset_rundir()
+    reset_server_log_file()
```

Comment:
@moscicki this is what we talked about in Berlin, where the before and after events would be awesome to have.
Any chance to get them in?